projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -8,22 +8,20 @@
-    "cy:run:ci:2005:b2b": "cypress run --config-file cypress.ci.b2b.json --record --key $CYPRESS_KEY --tag \"2005,b2b,all\" --spec \"cypress/integration/b2b/**/*.e2e-spec.ts\"",
+    "cy:run:ci:2005": "cypress run --config-file cypress.ci.2005.json --record --key $CYPRESS_KEY --tag \"2005,b2c,all,parallel\" --parallel --group $STAGE_NAME --spec \"cypress/integration/!(vendor|b2b)/**/*.e2e-spec.ts\"",
     "cy:run:ci:ccv2": "cypress run --config-file cypress.ci.ccv2.json --record --key $CYPRESS_KEY --tag \"ccv2,b2c,all\" --spec \"cypress/integration/!(vendor|b2b)/**/*.e2e-spec.ts\"",
+    "cy:run:ci:2005:b2b": "cypress run --config-file cypress.ci.b2b.json --record --key $CYPRESS_KEY --tag \"2005,b2b,all\" --parallel --group $STAGE_NAME --spec \"cypress/integration/b2b/**/*.e2e-spec.ts\"",
     "cy:run:mobile": "cypress run --spec \"cypress/integration/mobile/**/*\"",
     "cy:run:mobile:ci": "cypress run --config-file cypress.ci.1905.json --spec \"cypress/integration/mobile/**/*\"",
-    "cy:run:regression": "cypress run --spec \"cypress/integration/regression/**/*\"",
-    "cy:cds:run:vendor": "cypress run --spec \"cypress/integration/vendor/cds/**/*\"",
     "cy:cds:run:ci:2005": "cypress run --config-file cypress.ci.2005.json --record --key $CYPRESS_KEY --tag \"2005,b2c,all-cds\" --spec \"cypress/integration/!(vendor|b2b)/**/*.e2e-spec.ts,cypress/integration/vendor/cds/**/*.e2e-spec.ts\"",
     "cy:run:regression:ci": "cypress run --config-file cypress.ci.1905.json --spec \"cypress/integration/regression/**/*\"",
-    "cy:run:smoke": "cypress run --spec \"cypress/integration/smoke/**/*\"",
+    "cy:run:smoke": "cypress run --spec \"cypress/integration/smoke**/*\"",
     "cy:run:smoke:ci:1905": "cypress run --config-file cypress.ci.1905.json --record --key $CYPRESS_KEY --tag \"1905,b2c,smoke\" --spec \"cypress/integration/smoke/**/*\"",
     "cy:run:smoke:ci:2005": "cypress run --config-file cypress.ci.2005.json --record --key $CYPRESS_KEY --tag \"2005,b2c,smoke\" --spec \"cypress/integration/smoke/**/*\"",
     "cy:run:smoke:ci:2005:b2b": "cypress run --config-file cypress.ci.b2b.json --record --key $CYPRESS_KEY --tag \"2005,b2b,smoke\" --spec \"cypress/integration/b2b/smoke/**/*\"",
     "cy:run:smoke:ci:ccv2": "cypress run --config-file cypress.ci.ccv2.json --record --key $CYPRESS_KEY --tag \"ccv2,b2c,smoke\" --spec \"cypress/integration/smoke/**/*\"",
     "cy:run:b2b": "cypress run --config-file cypress.ci.b2b.json --spec \"cypress/integration/b2b/**/*\"",
-    "cy:run:b2b:ci": "cypress run --config-file cypress.ci.b2b.json --record --key $CYPRESS_KEY --tag \"2005,b2b,all\" --spec \"cypress/integration/b2b/**/*\"",
+    "cy:run:b2b:ci": "cypress run --config-file cypress.ci.b2b.json --record --key $CYPRESS_KEY --tag \"2005,b2b,all,parallel\" --spec \"cypress/integration/b2b/**/*\"",
```

Comment:
Should this job have tag `parallel`?

Reply:
It is not running because it makes the job fail and it is recorded as a separate run. But if it runs, yes, we want to parallelize the B2B tests as well